paparazzi/src/main/java/app/cash/paparazzi/internal/Renderer.kt

```diff
@@ -188,16 +188,7 @@ internal class Renderer(
   }
 
   private fun getNativeLibDir(): String {
-    val osName = System.getProperty("os.name").toLowerCase(Locale.US)
```

Comment:
to my earlier feedback, can we revert this method/Renderer, and instead move the osArch check to `Environment` under `checkInstalledJvm`?  My rationale is that it seems more appropriate to have this logic live in Environment, and leaving Renderer to assume things are set up correctly.

Reply:
my hope with this was to not just give a better error message but to actually make it work

Reply:
I get that, but I think trying to make it work can lead to other weird behaviors if the native lib artifact downloaded via dependency resolution isn't the expected one.

Reply:
thoughts?

Reply:
> can lead to other weird behaviors
Yeah fair, was trying to spend some time verifying if it would actually work or not, but hadn't reproduced the original issue, at least by running the tests in the repo. Will try to spend more time on if if I can, otherwise, not against your suggestion as a backup, at least makes the issue more clear.